manifests/image-references

```diff
@@ -0,0 +1,15 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  - name: cluster-platform-operators-manager
+    from:
+      kind: DockerImage
+      name: controller:latest
+  - name: kube-rbac-proxy
+    from:
+      kind: DockerImage
+      name: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  - name: olm-rukpak
+    from:
+      kind: DockerImage
+      name: quay.io/operator-framework/rukpak:latest
```

Comment:
Should we have a tagged version instead of latest (which I also believe doesn't exist)?